### PR TITLE
add new article about v2.0.1

### PR DESCRIPTION
**Pull request by namusyaka**
/cc @zzak @kgrz @olleolleolle

### CODE REVIEW

File: _posts/2018-02-19-sinatra-2.0.1.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+publish_date: Monday, February 19, 2018
+---
+
+I have just released Sinatra v2.0.1 and Mustermann v1.0.2.
+
+Before we begin, I have to thank everyone who contributed, helped test pre-releases, and continues to use and support the project.
+
+## Security Fix
+
+The rack-protection-v2.0.1 contains security fix about [CVE-2018-7212](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-7212).
```
Comment (olleolleolle) [inline _posts/2018-02-19-sinatra-2.0.1.markdown]: Grammar note: “a security fix”

File: _posts/2018-02-19-sinatra-2.0.1.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+* `sinatra`: v2.0.1
+* `sinatra-contrib`: v2.0.1
+* `rack-protection`: v2.0.1
+* `mustermann`: v1.0.2
+* `mustermann-contrib`: v1.0.2
+
+### sinatra, sinatra-contrib and rack-protection v2.0.1
+
+Find out what's new in 2.0.1 on [CHANGELOG.md](https://github.com/sinatra/sinatra/blob/v2.0.1/CHANGELOG.md#201--2018-02-17)
```
Comment (olleolleolle) [inline _posts/2018-02-19-sinatra-2.0.1.markdown]: Grammar “in CHANGELOG.md”

File: _posts/2018-02-19-sinatra-2.0.1.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+### mustermann and mustermann-contrib v1.0.2
+
+Find out what's new in 1.0.2 on [CHANGELOG.md](https://github.com/sinatra/mustermann/tree/v1.0.2#stable-releases)
```
Comment (olleolleolle) [inline _posts/2018-02-19-sinatra-2.0.1.markdown]: “in CHANGELOG.md”

File: _posts/2018-02-19-sinatra-2.0.1.markdown [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+layout: post
+title: Sinatra 2.0.1 is out!
+author: Kunpei Sakai
+author_url: https://github.com/namusyaka
+publish_date: Monday, February 19, 2018
+---
+
+I have just released Sinatra v2.0.1 and Mustermann v1.0.2.
+
+Before we begin, I have to thank everyone who contributed, helped test pre-releases, and continues to use and support the project.
```
Comment (olleolleolle) [inline _posts/2018-02-19-sinatra-2.0.1.markdown]: (Perhaps “I want to thank” sounds more positive?)

File: _posts/2018-02-19-sinatra-2.0.1.markdown [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+Before we begin, I want to thank everyone who contributed, helped test pre-releases, and continues to use and support the project.
+
+## Security Fix
+
+The rack-protection-v2.0.1 contains a security fix about [CVE-2018-7212](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-7212).
+
+We would highly recomend to upgrade your rack-protection.
+
+Also, the security fix has been backported as [v1.5.4](https://rubygems.org/gems/rack-protection/versions/1.5.4)
```
Comment (kgrz) [inline _posts/2018-02-19-sinatra-2.0.1.markdown]: Should we include something on the lines of "backported as v1.5.4 if you're on older sinatra versions" or something? Or is that self-evident?

Comment (namusyaka) [reply _posts/2018-02-19-sinatra-2.0.1.markdown]: Thanks for pointing this out. I've reflected your review.

**Pull request merged**